### PR TITLE
added note about updated security baseline

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/8389

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @ei, for your information

Thank you

File: windows/security/threat-protection/security-policy-settings/maximum-password-age.md [hunk 1]
```diff
@@ -39,6 +39,9 @@ The **Maximum password age** policy setting determines the period of time (in da
 
 Set **Maximum password age** to a value between 30 and 90 days, depending on your environment. This way, an attacker has a limited amount of time in which to compromise a user's password and have access to your network resources.
 
+> [!NOTE]
+> Security baseline recommended by Microsoft doesn't contain the password-expiration policy, as this mitigation is less effective than modern ones. However, companies that didn't implement Azure AD Password Protection, multifactor authentication or other modern mitigations of password-guessing attacks, should leave this policy effective.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/security-policy-settings/maximum-password-age.md:43]:
```suggestion
> The security baseline recommended by Microsoft doesn't contain the password-expiration policy, as it is less effective than modern mitigations. However, companies that didn't implement Azure AD Password Protection, multifactor authentication, or other modern mitigations of password-guessing attacks, should leave this policy in effect.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by illfated — APPROVED** [review]
Looks good to me.

**Comment by STO123456789** [comment]
I have already filled a complaint with the ICCC and the F.B.I. For violation of privacy and infringement 

Ernesto

**Comment by STO123456789** [comment]
So you know I do have an appointment with an attorney tomorrow to prosecute 

Ernesto

**Pull request merged**